Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasFunctionRegistry_Source.jinja

```diff
@@ -123,6 +123,10 @@ namespace ScriptCanvas
 {% endif %}
 {% if hasbranch|booleanTrue == true %}
                     ->Attribute(AZ::ScriptCanvasAttributes::BranchOnResult, branchResultInfo)
+{% endif %}
+{% set useCompactStyle = macros.CleanName(function.attrib['UseCompactStyle']) %}
+{% if useCompactStyle is defined and useCompactStyle.lower() == "true" %}
+                    ->Attribute(AZ::ScriptCanvasAttributes::UseCompactStyle, true)
 {% endif %}
```

Comment:
without seeing how you use the attribute, I wonder if it's even necessary when you could just have the template produce the necessary code. The risk of using attributes is that we lose the ability to change the templates and produce different code, instead we go back to the more traditional style in which we disconnect the code generation from what we produce.